requirements.txt

```diff
@@ -1,8 +1,7 @@
-ansible==8.4.0
 ansible-builder==3.0.0
-ansible-lint==6.19.0
-ansible-navigator==3.4.2
+ansible-lint==6.21.1
+ansible-navigator==3.5.0
 ansible-runner==2.3.4
 boto3==1.28.49
 botocore==1.31.49
-kubernetes==27.2.0
```

Comment:
i thought `kubernetes.core` needed this pip package?

Reply:
The fips-assessment collection pulls it in. As I stated above though, it should probably be left in.